contracts/smart-account/modules/StealthAddressValidatoinModules/StealthAddressRegistryModule.sol

```diff
@@ -0,0 +1,260 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import {BaseAuthorizationModule} from "../BaseAuthorizationModule.sol";
+import {EIP1271_MAGIC_VALUE} from "contracts/smart-account/interfaces/ISignatureValidator.sol";
+import {ECDSA} from "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import {UserOperation} from "@account-abstraction/contracts/interfaces/UserOperation.sol";
+import {StealthAggreagteSignature} from "./StealthAggreagteSignature.sol";
+
+/**
+ * @dev Storage structure for Stealth Address Registry Module.
+ * StealthPubkey, dhkey are used in aggregated signature.
+ * EphemeralPubkey is used to recover private key of stealth address.
+ */
+struct StealthStorage {
+    uint256 stealthPubkey;
+    uint256 dhkey;
+    uint256 ephemeralPubkey;
+    address stealthAddress;
+    uint8 stealthPubkeyPrefix;
+    uint8 dhkeyPrefix;
+    uint8 ephemeralPrefix;
+}
```

Comment:
Please create an interface `IStealthAddressRegistryModule` and move structs, events and errors to the interface

Reply:
Done